modules/ibm-z-configure-hw-based-cex-encryption.adoc

```diff
@@ -0,0 +1,162 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ibm_z/installing-ibm-z.adoc
+// * installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc
+// * installing/installing_ibm_z/installing-ibm-z-kvm.adoc
+// * installing/installing_ibm_z/installing-restricted-networks-ibm-z-kvm.adoc
+// * installing/installing_ibm_z/installing-ibm-z-lpar.adoc
+// * installing/installing_ibm_z/installing-restricted-networks-ibm-z-lpar.adoc
+
+ifeval::["{context}" == "installing-ibm-z"]
+:ibm-z:
+endif::[]
+ifeval::["{context}" == "installing-ibm-z-kvm"]
+:ibm-z-kvm:
+endif::[]
+ifeval::["{context}" == "installing-ibm-z-lpar"]
+:ibm-z-lpar:
+endif::[]
+ifeval::["{context}" == "installing-restricted-networks-ibm-z"]
+:ibm-z:
+endif::[]
+ifeval::["{context}" == "installing-restricted-networks-ibm-z-kvm"]
+:ibm-z-kvm:
+endif::[]
+ifeval::["{context}" == "installing-restricted-networks-ibm-z-lpar"]
+:ibm-z-lpar:
+endif::[]
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-luks-encryption-via-cex-ibm-z-linuxone-environment_{context}"]
+= Linux Unified Key Setup (LUKS) encryption via IBM Crypto Express (CEX) in an {ibm-z-title} or {ibm-linuxone-title} environment
+
```

Comment:
[ISG ](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#v)appears to discourage use of "via". Would it make sense to say something similar to the following?

"Enabling hardware-based LUKS encryption by using CEX in an {ibm-z-name} or {ibm-linuxone-name} environment requires..."

If this would necessitate a large change across many files, it might not be worth the effort.

Reply:
"using" doesn't really fit here CEX is a built in card on IBM Z

BTW 😏 I researched the repo yesterday because I know the ISG guidance. Search brought almost 2000 occurences 😆